netflix-roulette/cypress/integration/1-getting-started/movies.spec.js

```diff
@@ -0,0 +1,47 @@
+describe("Input form", () => {
+  it("opens the site", () => {
+    cy.visit("localhost:4200"); //this is the server that was started earlier
+  });
+
+  it("check input text", () => {
+    cy.get("input[type=search]").type("test").should("have.value", "test");
```

Comment:
It is recommended to [avoid ambiguous selectors and use `data-*` attributes](https://docs.cypress.io/guides/references/best-practices#Selecting-Elements) to identify tested elements.